api/SVGElement.json

```diff
@@ -348,42 +348,42 @@
           "spec_url": "https://svgwg.org/svg2-draft/interact.html#LoadEvent",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "12"
             },
             "firefox": {
-              "version_added": true,
+              "version_added": "4",
               "alternative_name": "SVGLoad"
```

Comment:
Does Firefox still not fire a "load" event? I expect this is wrong, that in fact "SVGLoad" is no longer being fired and there's just "load". A test for this can probably also be written using `<image>`.

Reply:
Surprisingly enough, it doesn't appear to, at least if we added the event listener to the `<svg>` or `<image>` elements rather than the document.

Reply:
But it does fire a "SVGLoad" event?

Reply:
Precisely, from Firefox 4 to present.

Reply:
Huh. Can you link https://bugzilla.mozilla.org/show_bug.cgi?id=620002 in a note so it's easy to figure this out the next time around?

Reply:
Thanks for finding the bug, added!